### ISSUE TITLE
Experiment Module: rename 'Related' button as 'Quick Links'

### ISSUE THREAD

**Issue by aislinnpearson** [Rothamsted-Ecoinformatics/farm_rothamsted#672]
When demonstrating the Related tab to one of our scientists, she mentioned it was not the best named button. She assumed it meant 'related experiments' where really it means 'related entities' or 'linked entities'. Another naming option might be 'Quick Links'. 

Marked this as 'effort: trivial' because it only relates to renaming the existing button. More broadly @paul121 mentioned we might want to also look at the user interface in the Plans and Templates work, to make it easy to see what other 'things' (assets, entities, plans, logs, etc) link to a specific 'thing'  (assets, entities, plans, logs, etc)

**Comment by mstenta** [comment]
https://openlayers.org/en/latest/examples/export-pdf.html

**Comment by aislinnpearson** [comment]
This looks great @paul121. I only have one minor request: on the plans can we swap the 'logs' and 'quick links' so the quick links are always at the end - just to remain consistent with the rest of the experiment module.

When checking I also realised there is no quick links for researchers. I think we did this deliberately, but I know people sometimes try to find proposals and experiments by researchers so I made an issue for adding them here: #729 

![image](https://github.com/user-attachments/assets/3fad5143-5e13-4643-84da-1b913f930949)
